### PR TITLE
[Build] Add test admin when starting gradle run with trial license

### PR DESCRIPTION
**Pull request by martijnvg**
and also add more documentation around gradle run task.

### CODE REVIEW

File: TESTING.asciidoc [hunk 1]
```diff
@@ -25,6 +25,46 @@ run it using Gradle:
...
+To for example start the open source distribution:
+
+-------------------------------------
+./gradlew run -Drun.distribution=oss-zip
+-------------------------------------
+
+==== License type
+
+By default a node is started with the `basic` license type.
+In order to start with a different license type use the `-Drun.distribution` argument.
```
Comment (rjernst) [inline TESTING.asciidoc]: `-Drun.distribution` -> `-Dlicense_type`

File: TESTING.asciidoc [hunk 2]
```diff
@@ -25,6 +25,46 @@ run it using Gradle:
...
+
+In order to start a node with a trail license execute the following command:
```
Comment (rjernst) [inline TESTING.asciidoc]: typo: trail -> trial

File: TESTING.asciidoc (hunk 1 above)
Comment (rjernst) [reply TESTING.asciidoc]: Maybe we should also change this to `run.license_type`?

**Review by rjernst — APPROVED** [review]
LGTM

File: TESTING.asciidoc (hunk 1 above)
Comment (martijnvg) [reply TESTING.asciidoc]: oops. Silly copy paste error. I will rename it to `run.license_type`.

**Pull request merged**